### PR TITLE
Adding style prop

### PR DESCRIPTION
**Pull request by giladartzi**
Please tell me if I need to add tests (and how to test - I'm not sure whether to add the style functionality to the driver of not)

### CODE REVIEW

**Comment by lbelinsk** [comment]
Use use window.getComputedStyle

**Comment by giladartzi** [comment]
Good idea - done

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.driver.ts [hunk 1]
```diff
@@ -24,6 +24,14 @@ export const toggleSwitchDriverFactory = ({element, componentInstance, eventTrig
     getToggleIcon: () => element.querySelector('.toggleIcon'),
     /** Returns the id of the input component */
     getId: () => element.querySelector('input').getAttribute('id'),
+    /** Returns the computed styles object of the root component */
+    getRootInlineStyles: () => window.getComputedStyle(element),
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.driver.ts]: This is not the Inline-style, but the general actual style that sits on this element.
so better to rename it to getOuterLabelStyles

File: packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.tsx [hunk 2]
```diff
@@ -11,11 +11,19 @@ type ToggleSwitchClasses = {
...
+  innerLabel: object;
+  toggleIcon: object;
+};
+
 interface ToggleSwitchProps {
   checked?: boolean;
   disabled?: boolean;
   onChange: React.EventHandler<React.ChangeEvent<HTMLInputElement>>;
   classes: ToggleSwitchClasses;
+  styles: ToggleSwitchStyles;
```
Comment (lbelinsk) [inline packages/wix-ui-core/src/components/ToggleSwitch/ToggleSwitch.tsx]: missing "?" since this is not mandatory

**Comment by lbelinsk** [comment]
will decide tomorrow in the meeting with @shlomitc whether to merge or not

**Pull request merged**